open_gpt/models/generation.py

```diff
@@ -336,4 +347,17 @@ def generate(self, prompt: str, max_context_length: int = CONTEXT_LEN, **kwargs)
                 skip_special_tokens=skip_special_tokens,
             )
 
-            return text
+            resp = {
+                "choices": [
+                    {"index": 0, "text": text, "finish_reason": "length"},
```

Comment:
shoud't the `finish_reason = 'stop'`

Reply:
`stop` only happens when we set the `stop_str` and it appears in generated text